builders/get-started/eth-compare/security.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Security Considerations
+description: A description of the main differences that Ethereum developers need to understand in terms of security considerations when developing on Moonbeam.
+---
+
+# Security Considerations
+
+![Moonbeam v Ethereum - Security Banner](/images/builders/get-started/eth-compare/security/security-banner.png)
+
+## Introduction {: #introduction }
+
+When developing smart contracts on Moonbeam, there are some security considerations to be aware of that do not apply when developing on Ethereum. Moonbeam has several [precompiled contracts](/builders/pallets-precompiles/precompiles/){target=_blank}, which are Solidity interfaces that enable developers to access Substrate-based functionality through the Ethereum API, but circumventing the EVM. Although the precompiled contracts are designed to improve the developer experience, there can be some unintended consequences that must be considered.
+
+This guide will outline and provide examples of some security considerations to be cognizant of when developing on Moonbeam.
+
+## Arbitrary Code Execution {: #arbitrary-code-execution }
+
+Arbitrary code execution in Solidity is the ability to execute code and call functions of other contracts using an arbitrary number of arguments of any type.
+
+A smart contract allows arbitrary execution of another contract when it allows a user to invoke the `call()` function and pass in arbitrary call data. The [`call()` function](https://solidity-by-example.org/call/){target=_blank} is made available through the [address data type in Solidity](https://docs.soliditylang.org/en/latest/types.html#address){target=_blank}. When the `call()` function is invoked, the target contract is called using the arbitrary call data.
```

Comment:
```suggestion
A smart contract allows arbitrary execution of another contract when it allows a user to influence its own `call()` and pass in arbitrary call data and/or the `call()`s target. The [`call()` function](https://solidity-by-example.org/call/){target=_blank} is made available through the [address data type in Solidity](https://docs.soliditylang.org/en/latest/types.html#address){target=_blank}. When the `call()` function is invoked, the target contract is called using the arbitrary call data.
```

Disregard or change as you want, what I'm getting at is:

* Clarity on which call is affected (it's Contract A in our example, not the EOA transaction call)
* Point out that issues can arise when the target is arbitrary/influence-able, it doesn't just have to be the call data